system-contract-dapp-playground/src/sections/hts-hip-206/index.tsx

```diff
@@ -0,0 +1,121 @@
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import Link from 'next/link';
+import { motion } from 'framer-motion';
+import { BsGithub } from 'react-icons/bs';
+import { Tooltip } from '@chakra-ui/react';
+import { IoOpenOutline } from 'react-icons/io5';
+import { HEDERA_SMART_CONTRACTS_ASSETS } from '@/utils/constants';
+
+const HTS206Section = () => {
+  return (
+    <motion.section
+      initial={{ opacity: 0 }}
+      whileInView={{ opacity: 1 }}
+      transition={{
+        delay: 0.3,
+        duration: 0.6,
+      }}
+      viewport={{ once: true }}
+      className="text-white w-full flex pt-7 pb-12 pl-7 flex-col gap-9"
+    >
+      {/* top part */}
+      <div className="flex flex-col gap-6">
+        {/* TITLE */}
+        <h1 className="text-[1.88rem] font-medium leading-10 flex gap-1">
+          HTS System Contract Wrapper
+          <div className="flex">
+            (
+            <Link
+              href={'https://hips.hedera.com/hip/hip-206'}
+              target="_blank"
+              className="hover:underline flex gap-/1 items-center"
+            >
+              <p>HIP-206</p>
+              <div className="text-2xl">
+                <IoOpenOutline />
+              </div>
+            </Link>
+            )
+          </div>
+        </h1>
+
+        {/* content */}
+        <div>
+          {/* Overview */}
+          <div className="text-[1.65rem] font-medium">Overview</div>
+
+          {/* content */}
+          <p className="tracking-tight text-white/70">
+            The integration of Hedera Token Service (HTS) with the Hedera Smart Contract Service
+            (HSCS), allowing contracts to transfer, mint, burn, associate, and dissociate tokens
+            programmatically.
+          </p>
+        </div>
+      </div>
+
+      {/* break line */}
+      <hr className="border-t border-white/40" />
+
+      {/* Smart contracts */}
+      <div className="flex flex-col gap-6">
+        {/* title & abstract*/}
+        <div>
+          {/* title */}
+          <h3 className="text-[1.65rem] font-medium">Example Smart Contracts</h3>
+
+          {/* abstract */}
+          <p className="tracking-tight text-white/70">
+            These example smart contracts demonstrate how the HTS system contract is employed to
+            carry out their functions effectively. They serve as clear examples of how the HTS
+            system contract can be put to practical use.
+          </p>
+        </div>
+
+        {/* Contracts */}
+        <ol className="px-6 flex flex-col gap-9">
+          {HEDERA_SMART_CONTRACTS_ASSETS.slice(0, 4).map((contract, index) => (
```

Comment:
Hmm `slice(0, 4)` could turn out to be problematic.
Essentially this requires the order of assets never change in the 1st 4 elements.
If it ever did the impact would be unclear.
Is there another constant that can hold what should be displayed. Maybe it feeds into `HEDERA_SMART_CONTRACTS_ASSETS`